lib/tasks/users.rake

```diff
@@ -0,0 +1,8 @@
+namespace :user do
+  desc 'Creating authorizations= for users'
+  task generate_auth: :environment do
+    User.all.each do |user|
```

Comment:
@marderer `find_each` can be used, see https://apidock.com/rails/ActiveRecord/Batches/ClassMethods/find_each